packages/rpc-engine/middlewares/validate-rpc-params.js

```diff
@@ -1,31 +1,35 @@
 import {defMiddleware} from '../middleware.js'
 import {validate, explain} from '@fluent-wallet/spec'
+import {addBreadcrumb} from '@fluent-wallet/sentry'
 
-export default defMiddleware(({tx: {map}}) => ({
+export default defMiddleware(({tx: {map, comp, sideEffect}}) => ({
   id: 'validateRpcParams',
   ins: {
     req: {stream: '/injectFetchFn/node'},
   },
-  fn: map(({rpcStore, req}) => {
-    const {params, method} = req
-    const {schemas, Err} = rpcStore[method]
-    if (schemas.input) {
-      if (!validate(schemas.input, params, {netId: req.network.netId})) {
-        throw Err.InvalidParams(
-          `input params:\n${JSON.stringify(
-            params,
-            null,
-            '\t',
-          )}\n\nError:\n${JSON.stringify(
-            explain(schemas.input, params, {netId: req.network.netId}),
-            null,
-            '\t',
-          )}`,
-          req,
-        )
+  fn: comp(
+    sideEffect(() => addBreadcrumb({category: 'validateRpcParams'})),
```

Comment:
I am worried about it is not safe that the RPC data will send to Sentry, especially the signature data

Reply:
The breadcrumb here is only here to label the to be sent error. 

It's like the trace of the error.

Only data specified in description of breadcrumbs will be sent.

Reply:
ok, maybe similar to that we will know the action path when debugging the error